Library/Homebrew/test/dev-cmd/extract_spec.rb

```diff
@@ -1,9 +1,10 @@
 # frozen_string_literal: true
 
 require "cmd/shared_examples/args_parse"
+require "dev-cmd/extract"
 
-RSpec.describe "brew extract" do
-  it_behaves_like "parseable arguments"
+RSpec.describe Homebrew::DevCmd::Extract do
+  it_behaves_like "parseable arguments", argv: ["foo", "bar"]
```

Comment:
Maybe we should just be able to pass the minimum number of required arguments instead of having to pass a garbage array every time?